When to call Page (#5034)
segmentio/segment-docs: src/connections/sources/catalog/libraries/website/javascript/index.md

Because some Destinations require a `page` call to instantiate their libraries, **you must call `page`** at least once per page load. You can call it more than once if needed, for example, on virtual page changes in a single page app.

> Please visit our implementation guide for more information on [when to call page()](https://segment.com/docs/getting-started/04-full-install/#when-to-call-page).

Reviewer: ```suggestion
See the implementation guide for more information about [calling the Page method](/docs/getting-started/04-full-install/#when-to-call-page).
```
